src/freon.c

```diff
@@ -29,6 +29,10 @@
     system("host-dbus dbus-send --system --dest=org.chromium.LibCrosService " \
```

Comment:
how about renaming this one to LIBCROSSERVICE_METHOD_CALL

Reply:
Done.